### PR TITLE
The Court Physician has arrived! +Bonuses!

### PR DESCRIPTION
**Pull request by Stagrawl**
## About The Pull Request

This PR adds in the Court Physician, a dedicated healer for the king and his men that doesn't require the church to loan out acolytes and also has some thematic tie-in with the space. 

Additionally, the Gravedigger joins the church official, becoming an devotee of Necra without having to bumrush the priest every round for a promotion and losing his job title in the process. 

## Why It's Good For The Game

One of my plans is to slowly expand out on various faith interactions and bring a little more of the gods (both humen and inhumen) to the world. Right now, a lot of flavour is lacking due to the absence of the faithful being around. 

As a side, this will reduce the demand for out of town clerics and the like. Hopefully we can slowly transfer a lot of what would be external travellers bringing power at a whim to a more consistent flow of towners serving specific roles within the community and actually having focused and dedicated duties to serve. 

The Gravedigger thing is largely a QoL change, but I think we'll see an improvement in the environment overall from having people accessing Necrite powers and being around actual corpses, something that doesn't often happen.

Bonus content: This adds craftable wooden psicrosses and finally utilises the wooden cross sprite.
Bonus fix: The magician's bag is longer full of heal + poison juice. We've scooped the reagents together and put them back into a bottle for him! 

I apologise if this is done a little messily, I lost all my progress at once point and had to essentially start from scratch.

image_2024-05-14_023539833

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/peasants/gravedigger.dm [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
+	allowed_patrons = list(/datum/patron/divine/necra)
 
 	allowed_races = list(
 		"Humen",
 		"Elf",
 		"Half-Elf",
 		"Dwarf",
-		"Tiefling",
-		"Argonian",
-		"Dark Elf",
```
Comment (BlueMemesauce) [inline code/modules/jobs/job_types/roguetown/peasants/gravedigger.dm:16]: bruh

we went from gravedigger being one of the only roles dark elf could be

to dark elf being banned from gravedigger

Comment (Stagrawl) [reply code/modules/jobs/job_types/roguetown/peasants/gravedigger.dm:16]: To be honest, I stripped this back to meet how other roles looked without much thought. I'd lean towards just throwing them back in tbh.

**Comment by YosemiteYam** [comment]
Gravedigger should not be in the church faction, they're supposed to be an outsider in society and looked down upon, hence why there's dark elves and shit in them. Putting them in the church makes them more respectable than what they really are and removes one more civilian profession from town and puts it in the ranks of the church.

**Comment by Real-MAGNUM** [comment]
Do not remove all races from grave digger. It's a KEY role for PQ salvaging.

Mapmerger wasnt used, safe to merge after discussion+review is done

**Comment by Stagrawl** [comment]
> Gravedigger should not be in the church faction, they're supposed to be an outsider in society and looked down upon, hence why there's dark elves and shit in them. Putting them in the church makes them more respectable than what they really are and removes one more civilian profession from town and puts it in the ranks of the church.

Gravediggers are already de-facto church members.


> Do not remove all races from grave digger. It's a KEY role for PQ salvaging.
> 
> Mapmerger wasnt used, safe to merge after discussion+review is done

Yeah, fully agree they should be added back in hindsight.

**Comment by YosemiteYam** [comment]
> > Gravedigger should not be in the church faction, they're supposed to be an outsider in society and looked down upon, hence why there's dark elves and shit in them. Putting them in the church makes them more respectable than what they really are and removes one more civilian profession from town and puts it in the ranks of the church.
> 
> Gravediggers are already de-facto church members.

They're not, they're a town member and deal with dead bodies. Proximity to church does not automatically make gravediggers holy, they do not work for the church they work for the betterment of everyone in the town by burying their loved ones and get paid for it. Making them just another job in the church just gets rid of another element of the town citizens and adds to church vs king bs by dividing the citizens into them. 

Besides why would gravediggers, aka a profession that requires money be IN the church? That implies that they are paid by the church as unlike acolytes and templars who are in the church after renouncing material gain the gravediggers need actual pay and thus how does the priest pay them money? Gravediggers aren't holy men hence this implies that the church has its own journeymen of a profession in them which makes no sense. If you are saying that they ARE holy men then now it means the church gets a bunch of new acolytes which again messes things up

**Comment by Stagrawl** [comment]
> > > Gravedigger should not be in the church faction, they're supposed to be an outsider in society and looked down upon, hence why there's dark elves and shit in them. Putting them in the church makes them more respectable than what they really are and removes one more civilian profession from town and puts it in the ranks of the church.
> > 
> > 
> > Gravediggers are already de-facto church members.
> 
> They're not, they're a town civilian role and deal with dead bodies. Proximity to church does not automatically make gravediggers holy, they do not work for the church they work for the betterment of everyone in the town by burying their loved ones and get paid for it. Making them just another job in the church just gets rid of another element of the town citizens and adds to church vs king bs by dividing the citizens into them.
> 
> Besides why would gravediggers, aka a profession that requires money be IN the church? That implies that they are paid by the church as unlike acolytes and templars who are in the church after renouncing material gain the gravediggers need actual pay and thus how does the priest pay them money? Gravediggers aren't holy men hence this implies that the church has its own journeymen of a profession in them which makes no sense. If you are saying that they ARE holy men then now it means the church gets a bunch of new acolytes which again messes things up as acolytes wouldn't be moonlighting as corpse-touchers and gives more spellcasters to the church.



You’re absolutely right, followers of Necra the Undermaiden Goddess (frequently worshipped by gravediggers!) wouldn’t be caught DEAD in a graveyard near corpses. Gross! That tree, well it’s just named that for some reason.

Such devout, spiritual figures especially wouldn’t lower themselves to performing such a lowly duty as BURIAL RITES just across from the church where all their high status(?) acolyte fellows dwell.

And of course they are so independent that the Priest just kindly offers to come and visit the graveyard with his GRAVEYARD KEYS as a favour, because the church has no interest in dead bodies at all.

**Comment by OrbitalParadise** [comment]
Bro what are u even saying

**Comment by OrbitalParadise** [comment]
This won't get merged anyway there is no medical system for it as of right now but you can keep working on it ig. Also chungus already made the apothecary which is exactly the same as this

**Comment by YosemiteYam** [comment]
> > > > Gravedigger should not be in the church faction, they're supposed to be an outsider in society and looked down upon, hence why there's dark elves and shit in them. Putting them in the church makes them more respectable than what they really are and removes one more civilian profession from town and puts it in the ranks of the church.
> > > 
> > > 
> > > Gravediggers are already de-facto church members.
> > 
> > 
> > They're not, they're a town civilian role and deal with dead bodies. Proximity to church does not automatically make gravediggers holy, they do not work for the church they work for the betterment of everyone in the town by burying their loved ones and get paid for it. Making them just another job in the church just gets rid of another element of the town citizens and adds to church vs king bs by dividing the citizens into them.
> > Besides why would gravediggers, aka a profession that requires money be IN the church? That implies that they are paid by the church as unlike acolytes and templars who are in the church after renouncing material gain the gravediggers need actual pay and thus how does the priest pay them money? Gravediggers aren't holy men hence this implies that the church has its own journeymen of a profession in them which makes no sense. If you are saying that they ARE holy men then now it means the church gets a bunch of new acolytes which again messes things up as acolytes wouldn't be moonlighting as corpse-touchers and gives more spellcasters to the church.
> 
> You’re absolutely right, followers of Necra the Undermaiden Goddess (frequently worshipped by gravediggers!) wouldn’t be caught DEAD in a graveyard near corpses. Gross! That tree, well it’s just named that for some reason.
> 
> Such devout, spiritual figures especially wouldn’t lower themselves to performing such a lowly duty as BURIAL RITES just across from the church where all their high status(?) acolyte fellows dwell.
> 
> And of course they are so independent that the Priest just kindly offers to come and visit the graveyard with his GRAVEYARD KEYS as a favour, because the church has no interest in dead bodies at all.

????

You're speaking gibberish at this point. Gravediggers are not Necra Acolytes, they're just gravediggers. Just as smiths aren't Malum acolytes, every warrior isn't a Ravox acolyte. People's jobs are not holy just because they serve a purpose or because they share one of the themes of a deity. What you propose would turn all gravediggers into Necra acolytes that will be able to cast Lesser Miracle, Churn Undead and Speak with Dead, effectively turning a job that has "Gravedigger" in its name and gravedigging as its main profession into being priests with all the spells they need to heal people and themselves, which isn't what they're here for. 

No matter how hard you want your headcanon to be true gravediggers do not work for the church just because the priest has keys to the graveyard. They are a town job that gets its own pay and in every RT iteration have been treated as the job that less-than-reputable people do because its not seen as clean/glorious/honorable. Yes dealing with dead bodies will make your brain go to acolytes/priests that want to revive or heal people before they become dead bodies, but that is where the similarity ends, just as a real life mortician is not the same thing as a real life doctor or priest, and they do not work for a church. We do not need to make the church a second garrison by lumping every role in it. 

I am unsure why you are so adamant on this issue for a pr that is mainly supposed to be about your rolebloat addition ie the court physician and the rest a "bonus". The court physician itself from what I can tell, only seems to be a renamed acolyte with no actual physician/surgery mechanic.

**Comment by Stagrawl** [comment]
> > > > > Gravedigger should not be in the church faction, they're supposed to be an outsider in society and looked down upon, hence why there's dark elves and shit in them. Putting them in the church makes them more respectable than what they really are and removes one more civilian profession from town and puts it in the ranks of the church.
> > > > 
> > > > 
> > > > Gravediggers are already de-facto church members.
> > > 
> > > 
> > > They're not, they're a town civilian role and deal with dead bodies. Proximity to church does not automatically make gravediggers holy, they do not work for the church they work for the betterment of everyone in the town by burying their loved ones and get paid for it. Making them just another job in the church just gets rid of another element of the town citizens and adds to church vs king bs by dividing the citizens into them.
> > > Besides why would gravediggers, aka a profession that requires money be IN the church? That implies that they are paid by the church as unlike acolytes and templars who are in the church after renouncing material gain the gravediggers need actual pay and thus how does the priest pay them money? Gravediggers aren't holy men hence this implies that the church has its own journeymen of a profession in them which makes no sense. If you are saying that they ARE holy men then now it means the church gets a bunch of new acolytes which again messes things up as acolytes wouldn't be moonlighting as corpse-touchers and gives more spellcasters to the church.
> > 
> > 
> > You’re absolutely right, followers of Necra the Undermaiden Goddess (frequently worshipped by gravediggers!) wouldn’t be caught DEAD in a graveyard near corpses. Gross! That tree, well it’s just named that for some reason.
> > Such devout, spiritual figures especially wouldn’t lower themselves to performing such a lowly duty as BURIAL RITES just across from the church where all their high status(?) acolyte fellows dwell.
> > And of course they are so independent that the Priest just kindly offers to come and visit the graveyard with his GRAVEYARD KEYS as a favour, because the church has no interest in dead bodies at all.
> 
> ????
> 
> You're speaking gibberish at this point. Gravediggers are not Necra Acolytes, they're just gravediggers. Just as smiths aren't Malum acolytes, every warrior isn't a Ravox acolyte. People's jobs are not holy just because they serve a purpose or because they share one of the themes of a deity. What you propose would turn all gravediggers into Necra acolytes that will be able to cast Lesser Miracle, Churn Undead and Speak with Dead, effectively turning a menial job that has "Gravedigger" in its name and gravedigging as its main profession into being priests with all the spells they need to heal people and themselves, which isn't what they're here for.
> 
> No matter how hard you want your headcanon to be true gravediggers do not work for the church just because the priest has keys to the graveyard. They are a town job that gets its own pay and in every RT iteration have been treated as the job that less-than-reputable people do because its not seen as clean/glorious/honorable. Yes dealing with dead bodies will make your brain go to acolytes/priests that want to revive or heal people before they become dead bodies, but that is where the similarity ends, just as a real life mortician is not the same thing as a real life doctor or priest, and they do not work for a church. We do not need to make the church a second garrison by lumping every role in it and we can keep our citizens as normies without making everyone some sort of spellcaster or amazing warrior. Some jobs are just that; normal jobs.
> 
> I am unsure why you are so adamant on this issue for a pr that is mainly supposed to be about your rolebloat addition ie the court physician and the rest a "bonus". The court physician itself from what I can tell, only seems to be a renamed acolyte with no actual physician/surgery mechanic.

Sure bro, we’ll see what the adults have to say I guess.

**Comment by ChungusGamer666** [comment]
Gravedigger changes invalidate necra acolytes and overall, not a fan.

**Comment by ChungusGamer666** [comment]
changes requested on discord

**Comment by Stagrawl** [comment]
This should be as requested.

**Comment by OrbitalParadise** [comment]
Bro removed drip and patrons from acolyte just to give it to archivist LMAO

**Comment by Stagrawl** [comment]
> Bro removed drip and patrons from acolyte just to give it to archivist LMAO

It was requested that they be taken out of the church as their presence doesn't make as much sense there.

**Comment by YosemiteYam** [comment]
Why would archivist get the moon wraps, they have unique archivist drip. Giving them noc acolyte wraps is just wierd. Maybe just put it in their room if they want to wear it or something.

**Comment by Real-MAGNUM** [comment]
Conflicting.

**Pull request closed without merging**